src/EditorFeatures/CSharp/CSharpEditorResources.resx

```diff
@@ -117,6 +117,18 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="Fix_misplaced_usings" xml:space="preserve">
+    <value>Fix misplaced usings</value>
+    <comment>{Locked="using"} "using" is a C# keyword and should not be localized.</comment>
+  </data>
+  <data name="Misplaced_using" xml:space="preserve">
+    <value>Misplaced using</value>
+    <comment>{Locked="using"} "using" is a C# keyword and should not be localized.</comment>
+  </data>
+  <data name="Move_misplaced_using_statements" xml:space="preserve">
+    <value>Move misplaced using statements</value>
+    <comment>{Locked="using"} "using" is a C# keyword and should not be localized.</comment>
```

Comment:
what is the difference from "Move misplaced" and "Fix misplaced"?

Reply:
This should be using directives, not statements right?